Model Validation Method

fsl506 opened the pull request:
Section will be written in this branch, then merged after reviewed successfully.

fsl506 committed:
Model Validation Method

Sakina-1 reviewed (approved):
Looks good to me. Maybe add how you can also make changes in the Python script, which is a lot quicker than using the OOF2 interface. I think this is good to merge, if anything needs to be added it shouldn't be too much of a hassle to add. Also checked the table, seems to match the test inputs correctly :)

fsl506 committed:
Image of OOF2 Tests Input

mtp19stl commented:
Looks good to me - happy to merge

mtp19stl reviewed (approved):
Looks good!

fsl506 committed:
Figure added/ref. generalised

fsl506 committed:
Formatting Improved

fsl506 committed:
FEM_TestCase1 added

fsl506 committed:
FEMTest1 Results and Discussion

fsl506 committed:
Title change

fsl506 committed:
Updated FEM subsection title

fsl506 committed:
Minor corrections to tense

fsl506 committed:
Simulation Method Added

Still missing the initial image of the microstructure used and the homogeneity index.

Gbowker commented:
Looks good. Happy for you to merge this in.

fsl506 committed:
Deleting file - uploading new simulation results

fsl506 committed:
Added FEM TestCase1

fsl506 committed:
Added FEMTestCase2_Temp PNG

fsl506 committed:
Added FEMTestCase3_Temp PNG

fsl506 committed:
Added FEMTestCase4_Temp PNG

fsl506 committed:
Updated Boundary Conditions

Altered boundary conditions as the simulations have been rerun.

fsl506 committed:
Delete FEMTestCase1

Correcting image to include mesh clearly in simulation result

fsl506 committed:
Added FEMTestCase1 - CORRECT

Mesh shown on simulation results

fsl506 committed:
Updated FEM TestCase1 Name in figure

fsl506 committed:
FEMTestCase2-4 Full Results PNGs

fsl506 committed:
Updated TestCase1 Figure Caption

fsl506 committed:
Added homogeneity index for final simulation

fsl506 committed:
Added TestCase2 Results

fsl506 committed:
Added TestCase3 Results

fsl506 committed:
TestCase4 Results Added

fsl506 committed:
Added Final Simulation Image + Formatting

fsl506 committed:
Initial Image for Final Simulation PNG Added

fsl506 commented:
I've finished writing up the model validation and simulation method for the FEM section. It should (fingers crossed) all be formatted correctly etc... I'm moving on to the results and discussion now.

fsl506 committed:
Grains changed to grain

mtp19stl reviewed (approved):
Looks good happy to merge

fsl506 committed:
Added PNG of Ti-64 Section Used

fsl506 committed:
Changed Final FEM Image/Caption

fsl506 committed:
Added test case 4 stress method

fsl506 committed:
Intro edit: project to programme

fsl506 committed:
Added stress contour method to final simulation

fsl506 committed:
Spelling Correction - Program

fsl506 committed:
Fixed micron unit errror

Sakina-1 reviewed (approved):
Looks good to me.
